### ISSUE TITLE
Add links to blog posts explaining erc721

### ISSUE THREAD

**Issue by come-maiz** [0xProject/erc721-website#24]
Here are two comprehensive posts with nice explanations about erc721:

https://medium.com/blockchannel/walking-through-the-erc721-full-implementation-72ad72735f3c
https://maksimivanov.com/posts/gradient-coin-tutorial/

I think it would be great to include them in this site.

**Comment by tomhschmidt** [comment]
Yeah, I'm a big fan of this idea. I'll merge the PR if someone wants to make it.

**Comment by fulldecent** [comment]
@tomhschmidt #31 is such a PR. It adds two entry level articles. And it leaves room to add more articles.

**Comment by abbaskiko** [comment]
working
